packages/tools/pie-css/package.json

```diff
@@ -7,11 +7,15 @@
     "index.css",
     "scss"
   ],
-  "type": "module",
-  "main": "index.css",
+  "main": "dist/index.css",
   "scripts": {
-    "lint:style": "run -T stylelint ./**/*.{css,scss} --allow-empty-input",
-    "lint:style:fix": "yarn lint:style --fix"
+    "build:css": "run -T ts-node ./build-css.ts",
```

Comment:
Would you mind renaming this to `build` just so it's consistent with the other packages?

Each time we deviate, it requires another entry into `turbo.json`, as well as CI changes as we currently only execute the `build` command. 👍🏼